### DISCUSSION TITLE
Shipping methods not showing on the checkout

### DISCUSSION THREAD

**Discussion by wrux** [swellstores/community#88] [Help] [answered]
I have a store ([link](https://www.browniemacs.co.uk/)) with a few simple shipping methods setup, however on the checkout the "shipping method" section is always empty. Is this a bug, or an issue with the shipping configuration? It looks like it should work and there's no errors logged in the browser console, so I'm not sure where to proceed. 

<img width="1231" alt="Screenshot 2023-01-16 at 20 01 47" src="https://user-images.githubusercontent.com/1453384/212673781-65f0c2b2-3592-4824-87af-ef66c7870132.png">

<img width="1187" alt="Screenshot 2023-01-16 at 20 02 45" src="https://user-images.githubusercontent.com/1453384/212673940-b65edc16-2212-4a22-91ef-a1d998e66055.png">

**Comment by awwit** [comment, marked as answer]
Hello @wrux

We fixed this issue some time ago.

To make this problem disappear, you can switch the main currency in the store to any other, and then return it back.

Or send a request to the store settings:

```
PUT /settings/shipments

Body: { "currency": "YOUR_CURRENCY_CODE" }
```
